pulp_deb/app/tasks/synchronizing.py

```diff
@@ -431,7 +439,7 @@ async def _handle_distribution(self, distribution):
             )
             await self.put(release_architecture_dc)
         # Parse release file
-        log.info('Parsing Release file for release: "{}"'.format(release_file.codename))
+        log.info('Parsing Release file for distribution: "{}"'.format(distribution))
```

Comment:
```suggestion
        log.info('Parsing Release file for release: "{}" at distribution: "{}"'.format(release_file.codename, distribution))
```
You never know which information might be really needed in the logfiles...

Reply:
Done.